pylearn2/models/mlp.py

```diff
@@ -3204,6 +3251,14 @@ def get_layer_monitoring_channels(self, state_below=None,
                            ('kernel_norms_max', row_norms.max()),
                            ])
 
+        if cudnn2d:
```

Comment:
I have to admit I have no idea. It was part of the original code so I merged it, but it doesn't seem to be of any use. I will remove it.